build-tools/docker/Dockerfile.node-daemon

```diff
@@ -1,16 +1,26 @@
 # Build Stage
 FROM mintlayer-builder:latest AS builder
 
-
 # Runtime Stage
 FROM debian:bookworm-slim
 
-COPY --from=builder /usr/src/target/release/node-daemon /usr/bin
+# Create a new user and group
+RUN groupadd -g 1000 mintlayer && \
+    useradd -u 1000 -g mintlayer -d /home/mintlayer -s /bin/bash -c "Mintlayer User" mintlayer
+
+# Create and set the home directory for the new user
+RUN mkdir /home/mintlayer && \
+    chown mintlayer:mintlayer /home/mintlayer
 
-# Node daemon listens on ports 13030 and 13031
-EXPOSE 13030 13031
+COPY --from=builder /usr/src/target/release/node-daemon /usr/src/target/release/wallet-cli /usr/bin/
```

Comment:
Since the "node-daemon" image now contains wallet-cli as well, should we rename it, e.g. to just "node"?

Reply:
I don't prefer renaming executables.

Reply:
I have concerns about combining node-daemon with wallet. There is a use case when wallet in not needed, eg for running api-server. Running two or more containers could be done via docker-compose

Reply:
I reverted this